generators/client/templates/angular/webpack/_webpack.dev.js

```diff
@@ -19,9 +49,13 @@ module.exports = webpackMerge(commonConfig({env: ENV}), {
         new BrowserSyncPlugin({
             host: 'localhost',
             port: 9000,
-            proxy: 'http://localhost:<%= serverPort %>'
+            proxy: 'http://localhost:9002'
```

Comment:
I think this one is proxying through webpack dev server so a fixed port might not be an issue its not the actual server port